pkg/config/cli.go

```diff
@@ -54,6 +55,7 @@ type ControllerOptions struct {
 	EnableDynamicConfig             bool
 	EnableMetricsScraper            bool
 	KubeControllerManagerExtraArgs  string
+	FeatureGates                    featuregate.FeatureGates
```

Comment:
I decided to add it here instead of adding it as yet another global variable such as `CfgFile`or `K0sVars`, this limits its capability to run as part of `k0s worker` but I'm not overly concerned about this at the moment.

I think this is cleaner and if we need to add it in the future we can worry about it later.